shell/platform/darwin/ios/framework/Source/FlutterTextInputPluginTest.m

```diff
@@ -240,6 +240,16 @@ - (void)testNoZombies {
   XCTAssert([[passwordView.textField description] containsString:@"TextField"]);
 }
 
+- (void)testInputViewCrash {
+  FlutterTextInputView* activeView = nil;
+  @autoreleasepool {
+    FlutterTextInputPlugin* inputPlugin = [FlutterTextInputPlugin new];
```

Comment:
This test is set up in a weird way, textInputDelegate is FlutterEngine, not input plugin.